src/crypter.js

```diff
@@ -47,6 +50,7 @@ module.exports.Crypter = (secret: Buffer, {
       let useGzip = false;
       if (gzip !== constants.GZIP_MODE.NEVER) {
         const inputGzip = zlib.gzipSync(buffer, { level: 9 /* zlib.constants.Z_BEST_COMPRESSION */ });
+        // how many cases will this actually be true (that the uncompressed buffer is actually less in size...?)
```

Comment:
In my testing quite a few. It really depends on what data is compressed and how long the data is.

Reply:
how much of a difference is there?